requirements.txt

```diff
@@ -1,4 +1,7 @@
 pydicom
 # This is a dcmstack branch with changes for Python 3
-git+git://github.com/ghisvail/dcmstack@master
+git+git://github.com/mvdoc/dcmstack@bf/importsys
```

Comment:
OK, however that repo (`github.com/ghisvail/dcmstack@master`) is not the main repository, which seems not maintained. But I did send a PR to this one.